README.md

```diff
@@ -0,0 +1,64 @@
+mender-dist-packages
+====================
+
+Mender is an open source over-the-air (OTA) software updater for embedded Linux devices. Mender comprises a client running at the embedded device, as well as a server that manages deployments across many devices.
+
+This repository contains mender-dist-packages, which is used to create distribution packages of the mender client software
+
+Currently only supports creation of a single `.deb` for `arm` arquitecture suitable for Raspberry Pi 3.
+
+![Mender logo](https://mender.io/user/pages/resources/06.digital-assets/mender.io.png)
+
+## Getting started
+
+To start using Mender, we recommend that you begin with the Getting started
+section in [the Mender documentation](https://docs.mender.io/).
+
+## Requirements
+
+You need to [install Docker Engine](https://docs.docker.com/install) to use this environment.
+
+### Instructions
+
+To build the container based on Debian Stretch:
+
+```bash
+docker build --tag=mender-dist-packages .
+```
+
+Create a directory output:
+
+```bash
+mkdir output
+```
+
+Create the deb package:
+
+```bash
+docker run --mount type=bind,source=$(pwd)/output,target=/output --privileged=true mender-dist-packages
```

Comment:
* I think you don't need `--privileged` (is it because of the `--mount`?) if you use volume instead:
  ```
  docker run -v $(pwd)/output:/output mender-dist-packages
  ```
* I would add `--rm` as well, so that the container is disposed of afterwards.
* Would it make sense to put this in a `build-deb` script?

Reply:
>     * I think you don't need `--privileged` (is it because of the `--mount`?) if you use volume instead:
>       ```
>       docker run -v $(pwd)/output:/output mender-dist-packages
>       ```
> 
Yes, it was because of the mount. Changed now
>     * I would add `--rm` as well, so that the container is disposed of afterwards.
>
Great, I didn't know about this flag!
>     * Would it make sense to put this in a `build-deb` script?
Soon we will need to modify this a bit to do several packages and probably to support different architectures, and then it would definitely make sense to wrap everything in an script. IMO for now (single manually created deb package) live with the instructions.